setup.sh

```diff
@@ -1,328 +1,15 @@
 #!/usr/bin/env bash
 
+. $(dirname $0)/lib/functions.sh
+. $(dirname $0)/lib/binaries.sh
```

Comment:
We can use some bash features to make this safer:
```bash
unset CD_PATH
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
cd "${SCRIPT_DIR}" || exit 1

source ./lib/functions.sh
source ./lib/binaries.sh
```

Reply:
Thanks, it's a nice way to set a predictable working directory for a script. Updated to match the above.